docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -31,94 +46,194 @@ It applies to Intel Core Ultra and Core 12 - 14 gen integrated GPUs (iGPUs), as
    
-   tokenizer = AutoTokenizer.from_pretrained("microsoft/phi-1_5", trust_remote_code=True)
+   print('Now start loading Tokenizer and Model...')
+   tokenizer = AutoTokenizer.from_pretrained("Qwen/Qwen-1_8B-Chat", trust_remote_code=True)
    # load Model using bigdl-llm and load it to GPU
    model = AutoModelForCausalLM.from_pretrained(
-       "microsoft/phi-1_5", load_in_4bit=True, cpu_embedding=True, trust_remote_code=True)
+       "Qwen/Qwen-1_8B-Chat", load_in_4bit=True, cpu_embedding=True, trust_remote_code=True)
    model = model.to('xpu')
+   print('Successfully loaded Tokenizer and Model!')
+
+   print('--------------------------------------Note----------------------------------------')
+   print('| When running LLMs on iGPU or Intel Arc™ A300-Series or Pro A60 for first time, |')
+   print('| it may take several minutes for GPU kernels to compile and initialize.         |')
+   print('| Please be patient until it finishes warm-up...                                 |')
+   print('----------------------------------------------------------------------------------')
 
    # Format the prompt
    question = "What is AI?"
-   prompt = " Question:{prompt}\n\n Answer:".format(prompt=question)
+   prompt = "user: {prompt}\n\nassistant:".format(prompt=question)
    # Generate predicted tokens
    with torch.inference_mode():
        input_ids = tokenizer.encode(prompt, return_tensors="pt").to('xpu')
-       # warm up one more time before the actual generation task for the first run, see details in `Tips & Troubleshooting`
-       # output = model.generate(input_ids, do_sample=False, max_new_tokens=32, generation_config = generation_config)
+       
+       # To achieve optimal and consistent performance, we recommend a one-time warm-up by running `model.generate(...)` an additional time before starting your actual generation tasks.
+       # If you're developing an application, you can incorporate this warm-up step into start-up or loading routine to enhance the user experience.
+       output = model.generate(input_ids, do_sample=False, max_new_tokens=32, generation_config = generation_config) # warm-up
+
        output = model.generate(input_ids, do_sample=False, max_new_tokens=32, generation_config = generation_config).cpu()
        output_str = tokenizer.decode(output[0], skip_special_tokens=True)
        print(output_str)
    ```
-   > Note: when running LLMs on Intel iGPUs with limited memory size, we recommend setting `cpu_embedding=True` in the `from_pretrained` function.
-   > This will allow the memory-intensive embedding layer to utilize the CPU instead of GPU.
+  ```eval_rst
+  .. note::
+  
+     ``from_pretrained`` will download model from Hugging Face by default; You may also set ``model_hub='modelscope'`` in ``from_pretrained`` to use ModelScope.
+     If you set ``model_hub='modelscope'`` in ``from_pretrained``, please first run ``pip install modelscope==1.11.0`` in Anaconda Prompt to install ModelScope.
+  ```
```

Comment:
Maybe change the code section to two tabs: Model from HF and Model from ModelScope

Reply:
For the modelscope code, please mention the `AutoTokenizer` imported from `modelscope`: https://github.com/intel-analytics/BigDL/blob/main/python/llm/example/GPU/ModelScope-Models/generate.py

Please also notice the user that the repo id on ModelScope may be difference from HF for some models.